archetype/engine-v2/src/main/java/io/helidon/build/archetype/engine/v2/spi/TemplateEngine.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.build.archetype.engine.v2.spi;
```

Comment:
Can you use the main package of this module. I.e `io.helidon.build.archetype.engine.v2`. I know that the Helidon guidelines mandate the use of a sub-package for the SPI, however none of the SPIs in the build-tools repository don't follow that rule.